anta/tests/mlag.py

```diff
@@ -1,93 +1,233 @@
 
 
 class VerifyMlagInterfaces(AntaTest):
     """
-    Verifies there are no inactive or active-partial MLAG interfaces.
+    This test verifies there are no inactive or active-partial MLAG ports.
+
+    Expected Results:
+        * success: The test will pass if there are NO inactive or active-partial MLAG ports.
+        * failure: The test will fail if there are inactive or active-partial MLAG ports.
+        * skipped: The test will be skipped if MLAG is 'disabled'.
     """
 
     name = "VerifyMlagInterfaces"
-    description = "Verifies there are no inactive or active-partial MLAG interfaces."
+    description = "This test verifies there are no inactive or active-partial MLAG ports."
     categories = ["mlag"]
     commands = [AntaTestCommand(command="show mlag", ofmt="json")]
 
     @AntaTest.anta_test
     def test(self) -> None:
-        """Run VerifyMlagInterfaces validation"""
+        """
+        Run VerifyMlagInterfaces validation
+        """
 
         command_output = cast(Dict[str, Dict[str, Any]], self.instance_commands[0].output)
 
         if command_output["state"] == "disabled":
             self.result.is_skipped("MLAG is disabled")
-        elif command_output["mlagPorts"]["Inactive"] != 0 or command_output["mlagPorts"]["Active-partial"] != 0:
-            self.result.is_failure(f"MLAG status is not OK: {command_output['mlagPorts']}")
-        else:
+            return
+
+        if command_output["mlagPorts"]["Inactive"] == 0 and command_output["mlagPorts"]["Active-partial"] == 0:
             self.result.is_success()
+        else:
+            self.result.is_failure(f"MLAG status is not OK: {command_output['mlagPorts']}")
 
 
 class VerifyMlagConfigSanity(AntaTest):
     """
-    Verifies there are no MLAG config-sanity inconsistencies.
+    This test verifies there are no MLAG config-sanity inconsistencies.
+
+    Expected Results:
+        * success: The test will pass if there are NO MLAG config-sanity inconsistencies.
+        * failure: The test will fail if there are MLAG config-sanity inconsistencies.
+        * skipped: The test will be skipped if MLAG is 'disabled'.
     """
 
     name = "VerifyMlagConfigSanity"
-    description = "Verifies there are no MLAG config-sanity inconsistencies."
+    description = "This test verifies there are no MLAG config-sanity inconsistencies."
     categories = ["mlag"]
     commands = [AntaTestCommand(command="show mlag config-sanity", ofmt="json")]
 
     @AntaTest.anta_test
     def test(self) -> None:
-        """Run VerifyMlagConfigSanity validation"""
+        """
+        Run VerifyMlagConfigSanity validation
+        """
 
         command_output = cast(Dict[str, Dict[str, Any]], self.instance_commands[0].output)
 
-        if "mlagActive" not in command_output.keys():
-            self.result.is_error("Incorrect JSON response - mlagActive state not found")
-        elif command_output["mlagActive"] is False:
+        if command_output["mlagActive"] is False:
```

Comment:
Why is this gone:

```
if "mlagActive" not in command_output.keys():
            self.result.is_error("Incorrect JSON response - mlagActive state not found")
```

I guess it was certainly there for some corner case we may not be able to repro today but we shoudl keep it